website/docs/tutorials/04-configuration-file.md

```diff
@@ -35,65 +38,98 @@ network = "testnet"
 
 
 ### `[project]`
+The `[project]` section of the `protostar.toml` file allows you to specify global options and settings for your project.
 #### `protostar-version`
-This attribute is used to warn you about compatibility issues between Protostar and a project.
+This attribute is used to manifest what Protostar version should be used with your project.
+It should be set to the latest compatible Protostar version.
+If you try to use a different version of Protostar with your project, you will receive a warning and you may experience unexpected errors.
 
 #### Shared Configuration
-You can keep command arguments in the `[project]` to share them across commands. Read the [`[COMMAND]` section](#command) to learn how to configure command arguments in the configuration.
+You can specify options that are shared by multiple Protostar commands in the `[project]` section.
+For example, the [`lib-path`](/docs/cli-reference#--lib-path-path) option is used by the [`install`](/docs/cli-reference#install), [`update`](/docs/cli-reference#update), and [`remove`](/docs/cli-reference#remove) commands, and [`cairo-path`](/docs/cli-reference#--cairo-path-path) is used by the [`build`](/docs/cli-reference#build) and [`test`](/docs/cli-reference#test) commands.
 
 ### `[contracts]`
-This section allows you to refer to a contract by a contract name across Protostar features. You can also combine many files into one contract.
+The `[contracts]` section allows you to define contract names and the corresponding files that make up each contract.
+This is useful because it allows you to refer to contracts by name rather than having to specify the full file path each time.
+You can also combine multiple files into a single contract.
+Currently, the `[contracts]` section is required by the [`protostar build`](/docs/cli-reference#build) command.
+
+Here is an example of how to define a contract in the [contracts] section:
+
 ```toml
 [contracts]
 my_contract = ["src/feature_a.cairo", "src/feature_b.cairo"]   
 ```
 
-:::note
-Currently, [`protostar build`](/docs/cli-reference#build) requires this section.
-:::
-
 ### `[COMMAND]`
-Configure command arguments, to avoid passing them manually. CLI can be represented in the configuration file in the following way:
+The `[COMMAND]` section of the `protostar.toml` file allows you to specify options and settings for a specific Protostar command.
 
-```console title="CLI"
-protostar format src tests --ignore-broken
+To configure command arguments in the `protostar.toml` file, you can specify them in a `[COMMAND]` section using the following format:
+
+```toml
+[COMMAND]
+flag = true
+list = [1, 2, 3]
+path = "./src"
```

Comment:
This is not very useful snippet, can we just go straight to the snippet for a specific command?

Reply:
[remove generic examples](https://github.com/software-mansion/protostar/pull/1308/commits/ef5d3606ed5ac0d3c44c6462ac4fc81448c27de3)